variant-remapping-automation/get_custom_assembly.py

```diff
@@ -126,6 +127,39 @@ def get_contig_accessions_in_fasta(fasta_path):
                     written_contigs.extend(match.findall(line))
         return written_contigs
 
+    @staticmethod
+    def rewrite_changing_names(input_fasta, output_fasta, contig_to_rename):
+        with open(input_fasta) as open_input, open(output_fasta, 'w') as open_output:
+            for line in open_input:
+                if line.startswith('>'):
+                    contig_name = line.split()[0][1:]
+                    if contig_name in contig_to_rename:
+                        line = '>' + contig_to_rename[contig_name] + '\n'
+                open_output.write(line)
+
+    @cached_property
+    def contig_names_in_fasta(self):
+        return self._get_contig_accessions_in_fasta(self.assembly_fasta_path)
+
+    @cached_property
+    def contig_to_rename(self):
+        rename_map = {}
+        genbank_contigs = []
+        set(row['GenBank-Accn'] for row in self.assembly_report_rows)
```

Comment:
Do you need this for anything?

Reply:
Nope... 
I guess I should remove it then ...